Consul.AspNetCore.Test/ServiceCollectionExtensionsTest.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Xunit;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Consul.AspNetCore.Test
+{
+	public class ServiceCollectionExtensionsTest
+	{
+		private readonly IServiceCollection services;
```

Comment:
The naming convention is '_lowerCamelCase' for private fields in the library. Can you please start the variable name with underscore?